frontend/src/container/NewDashboard/DashboardDescription/index.tsx

```diff
@@ -63,6 +67,30 @@ interface DashboardDescriptionProps {
 	handle: FullScreenHandle;
 }
 
+function removeValuesFromVariables(
```

Comment:
rename the function as it is not just removing variables but also uuid as well, will be confusing when revisiting this.

Reply:
changed to `sanitizeDashboardData`